docs/examples/index.js

```diff
@@ -449,6 +484,9 @@ themeManager.externalizePopover({
   }
 // Display the chart using the configured theme and data.
 myChart.setOption(themeManager.getChartOptions());
+
+// May need to adapt chart size according to last features displayed
+window.setTimeout(myChart.resize);
```

Comment:
This change should be removed after merge of https://github.com/Orange-OpenSource/ods-charts/pull/250

Reply:
done